### PR TITLE
loader,crypto: fix link error for crypto/internal/boring/sig.Standard…

### PR DESCRIPTION
**Pull request by dgryski**
…Crypto

### CODE REVIEW

File: src/crypto/internal/boring/sig/sig_other.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Package sig stubs internal/boringcrypto/sig
```
Comment (joeshaw) [inline src/crypto/internal/boring/sig/sig_other.go]: Nit: should be `crypto/internal/boring/sig`

File: src/crypto/internal/boring/sig/sig_other.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+package sig
+
+// BoringCrypto indicates that the BoringCrypto module is present.
+func BoringCrypto() {
+
```
Comment (joeshaw) [inline src/crypto/internal/boring/sig/sig_other.go:6]: Should we panic here and in `FIPSOnly`?

**Review by joeshaw — COMMENTED** [review]
👍

File: src/crypto/internal/boring/sig/sig_other.go (hunk 2 above)
Comment (dgryski) [reply src/crypto/internal/boring/sig/sig_other.go:6]: These functions aren't called; they're here so tooling can look for the body of the symbols and then figure out stuff about the binary: https://github.com/golang/go/blob/master/src/crypto/internal/boring/sig/sig_amd64.s#L7

(edit: ok, they're called, but have effectively empty bodies)

File: src/crypto/internal/boring/sig/sig_other.go (hunk 1 above)
Comment (dgryski) [reply src/crypto/internal/boring/sig/sig_other.go]: Fixed and repushed.

**Comment by deadprogram** [comment]
@dgryski does this branch perhaps need to be rebased against latest `dev`?

**Comment by dgryski** [comment]
Redid the Mac job; passed this time.

**Comment by dgryski** [comment]
Rebased anyway.

**Comment by aykevl** [comment]
While this is a fix, and it seems fine by me, this issue will also be fixed by #3103.

**Comment by dgryski** [comment]
I'm happy to close this in favour of the more general fix.

**Comment by joeshaw** [comment]
@dgryski The caveats in #3103 (currently Linux only, depending on a few other PRs to land first) make me think that this PR would be beneficial to land as a stopgap to prevent the regression in crypto package usage from 1.18 to 1.19 caused by this issue unless that other PR is likely to land for the next release.

**Comment by dgryski** [comment]
I suppose we could merge this in the mean time and revert it when the full solution lands.

**Review by aykevl — APPROVED** [review]
> I suppose we could merge this in the mean time and revert it when the full solution lands.

Seems fine by me.

File: src/crypto/internal/boring/sig/sig_other.go (hunk 2 above)
Comment (aykevl) [reply src/crypto/internal/boring/sig/sig_other.go:6]: Yeah, this is what made me hesitate a bit with this patch. It breaks these signatures. But apparently it's only used for amd64 which makes me a bit less nervous about it (and amd64 isn't even the primary target of TinyGo).

**Comment by deadprogram** [comment]
Thank you @dgryski for the fix and @aykevl for review.

Now merging.

**Pull request merged**